src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelTableRowEditorViewImpl.java

```diff
@@ -309,4 +309,9 @@ public void clearMaxListLengthError() {
     this.maxListLengthHelp.setVisible(false);
     this.maxListLengthHelp.setText("");
   }
+
+  @Override
+  public void setFacetSortConfig(FacetSortConfigViewEnum viewForConfig) {
+    // not yet implementing in GWT-based editor
```

Comment:
Will likely never edit in GWT-based version.  Implemented in the react column schema editor